rock_build

```diff
@@ -283,6 +290,9 @@ echo "Finished preparing, starting build at `date` with $RUBY"
   if [ -n "$USE_OCL" ]; then
       sed -i "s/USE_OCL:.*$/USE_OCL: $USE_OCL/g" ./autoproj/config.yml
   fi
+  if test "x$BUILD_TESTS" = "xtrue"; then
+      echo "autoproj_test_utility_default: true"  >> ./autoproj/config.yml
```

Comment:
This is private stuff, and can therefore break whenever I feel like breaking it. Use the public interface

~~~
autoproj test --enable
~~~
~~~